aries/aries_vcx_wallet/src/wallet/indy/mod.rs

```diff
@@ -104,16 +104,9 @@ impl IndySdkWallet {
     async fn search(
         &self,
         category: RecordCategory,
-        search_filter: Option<SearchFilter>,
+        search_filter: Option<String>,
```

Comment:
same

Reply:
This is intentional, I can open it separately if you prefer.